### ISSUE TITLE
`no-legacy-test-waiters` incorrectly flagging registerHelper import

### ISSUE THREAD

**Issue by jaydgruber** [ember-cli/eslint-plugin-ember#769]
I have a file that exports a `registerHelper` that `no-legacy-test-waiters` is flagging.
I only `import { registerHelper } from '@ember/test';`, no waiter involved.

I think this is running into a situation where:
```javascript
testWaitersIdentifiers.includes(node.callee.name)

> [undefined, undefined].includes(undefined) // true
```
Will push up a failing test case shortly

**Comment by scalvert** [comment]
Thanks for catching this, @jaydgruber. And thanks for the fix, @bmish.
